packages/fhir-router/src/fhirrouter.ts

```diff
@@ -143,31 +183,43 @@ async function patchResource(req: FhirRequest, repo: FhirRepository): Promise<Fh
   return [allOk, resource];
 }
 
+export type RestInteraction =
```

Comment:
Low pri mental note: we may want to try to merge this with `packages/server/src/util/auditevent.ts`, either here in `fhir-router` or in `core`.